silverback/_cli.py

```diff
@@ -102,6 +115,17 @@ def run(cli_ctx, account, runner_class, recorder_class, max_exceptions, path):
                 option_name="network", message="Network choice cannot support running app"
             )
 
+    if path is None:
+        path = Path.cwd() / "bots" / "bot.py"
```

Comment:
This should be the default if it is none
```suggestion
        path = Path.cwd() / "bot.py"
```